app/src/main/res/menu/menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/follow_user_search_button"
+        android:icon="@android:drawable/ic_menu_search"
+        android:title="User Search"
+        app:showAsAction="always" />
+</menu>
```

Comment:
This `menu` tag allows us to add an `item` to the action bar, which in our case is the user search button